static/js/tools/shared/vis_tools/vis_tool_instructions_box.tsx

```diff
@@ -53,59 +54,75 @@ function getPlaceInstructionToShow(props: VisToolInstructionsBoxProps): string {
   }
 }
 
+/**
+ * Helper function to determine which variation on the instructions for
+ * selecting statistical variables the box should show.
+ *
+ * Returns both desktop and mobile versions of the message in the format
+ *    [ desktop, mobile ]
+ *
+ * @param props props passed into VisToolInstructionsBox
+ * @returns i18n formatted strings to display in the instructions
+ */
+function getVariableInstructionToShow(visType: VisType): [string, string] {
+  switch (visType) {
+    case "map": {
+      return [
+        intl.formatMessage(toolMessages.infoBoxInstructionsVariableDesktopMap),
+        intl.formatMessage(toolMessages.infoBoxInstructionsVariableMobileMap),
+      ];
```

Comment:
this seems rather brittle - even though it's just for two, it would be more maintanable to have this in a dictionary with explicit keys